todo_app/e2etests/test_e2e.py

```diff
@@ -12,10 +12,20 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
-@pytest.fixture(scope="module")
+#@pytest.fixture(scope="module")
+#def driver():
+#    driver = webdriver.Firefox()
+#    return driver
```

Comment:
We should delete this, or at the very least add a note at the top saying it's an alternative for running the tests locally/outside Docker